Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_incidents_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    fromdate = args.get("fromdate")
+    todate = args.get("todate")
+    status = args.get("status")
+    adversary_types = args.get("adversary-types")
+    labels = args.get("labels")
+
+    if status and isinstance(status, str):
+        status = status.split(",")
+    if adversary_types and isinstance(adversary_types, str):
+        adversary_types = adversary_types.split(",")
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_incidents_request(
+        page, items, fromdate, todate, status, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveIncidents",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_a_specific_incident_details_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+
+    response = client.retrieve_a_specific_incident_details_request(lumu_incident_id)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificIncidentDetails",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_a_specific_incident_context_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    hash = args.get("hash")
+
+    response = client.retrieve_a_specific_incident_context_request(
+        lumu_incident_id, hash
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificIncidentContext",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def comment_a_specific_incident_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    lumu_incident_id = args.get("lumu_incident_id")
+    comment = generate_hmac_sha256_msg(
+        client.api_key, add_prefix_to_comment(args.get("comment"))
+    )
+    try:
+
+        response = client.comment_a_specific_incident_request(lumu_incident_id, comment)
+    except DemistoException as err:
+
+        error_msg = "Failed to parse json object from response: b''"
+        if str(err) == error_msg:
+            response = {"statusCode": 200}
+        else:
+            raise DemistoException(err)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.CommentASpecificIncident",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
```

Comment:
Add `readable_output` to all the commands
```suggestion
        raw_response=response,
        readable_output='Comment added to the incident successfully.',
```

Reply:
readable_output was set in commands in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7